recipes/openbabel/2.3.2_7d621d9/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
+  name: openbabel
+  version: 2.3.2_7d621d9
+
+source:
+  url: https://github.com/openbabel/openbabel/archive/7d621d9c9f2f07a1f105896a765bf2afa22e6028.zip
+  fn: 7d621d9c9f2f07a1f105896a765bf2afa22e6028.zip
+
+  patches:
+    - include-dirs.patch
+    - fix_library_path_search.diff
+    - fix_data_path.diff
+
+build:
+  number: 0
+  detect_binary_files_with_prefix: true
+  skip:
+    - [win]
+
+requirements:
+  build:
+    - cmake
+    - zlib
+    - swig
+    - eigen
+    - bzip2
+    - python
+    - zlib
+    - libxml2
```

Comment:
That's a great idea. One way is to remove gcc and llvm from the docker container.